feature-cancel-insurance/src/main/kotlin/com/hedvig/android/feature/cancelinsurance/navigation/CancelInsuranceDestinations.kt

```diff
@@ -0,0 +1,68 @@
+package com.hedvig.android.feature.cancelinsurance.navigation
+
+import androidx.compose.material3.windowsizeclass.WindowSizeClass
+import androidx.navigation.NavGraphBuilder
+import androidx.navigation.NavHostController
+import androidx.navigation.navOptions
+import com.hedvig.android.feature.cancelinsurance.CancelInsuranceViewModel
+import com.hedvig.android.feature.cancelinsurance.InsuranceId
+import com.hedvig.android.feature.cancelinsurance.ui.result.CancellationSuccessDestination
+import com.hedvig.android.feature.cancelinsurance.ui.terminationdate.TerminationDateDestination
+import com.kiwi.navigationcompose.typed.Destination
+import com.kiwi.navigationcompose.typed.composable
+import com.kiwi.navigationcompose.typed.createRoutePattern
+import com.kiwi.navigationcompose.typed.navigate
+import com.kiwi.navigationcompose.typed.navigation
+import com.kiwi.navigationcompose.typed.popUpTo
+import kotlinx.serialization.Serializable
+import org.koin.androidx.compose.koinViewModel
+import org.koin.core.parameter.parametersOf
+
+internal sealed interface Destinations : Destination {
+  @Serializable
+  object CancelInsurance : Destinations
+}
```

Comment:
This is like a route to the entire feature. In such a future, there'd be many destinations globally available where you can navigate from any other point in the app.
So navigating to this is like saying "Go to the terminate insurance feature, and it will handle its own navigation internally"

Reply:
Super cool